specs/AppLifecycle/EnvironmentVariables/AppLifecycle Environment Variables.md

```diff
@@ -293,8 +295,6 @@ are listed in the table below:
 | Unpackaged     | Managed  | Read, Write  | Read, Write | Read, Write  |
```

Comment:
Line 286: "They can also use the managed System.Environment class..."

SUGGESTION: "Unpackaged .NET apps can use [System.Environment](https://docs.microsoft.com/dotnet/api/system.environment)..."

SUGGESTION: Update `GetEnvironmentVariable` and other API references to hyperlinks to their Win32 API docs